src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Commands/SetAzureRmDataLakeStoreItemAclEntry.cs

```diff
@@ -56,7 +56,7 @@ public class SetAzureDataLakeStoreItemAclEntry : DataLakeStoreFileSystemCmdletBa
                 "The ACL spec containing the entries to set. These entries MUST exist in the ACL spec for the file already. This can be a modified ACL from Get-AzureDataLakeStoreItemAcl or it can be the string " +
                 " representation of an ACL as defined in the apache webhdfs specification. Note that this is only supported for named ACEs." +
                 "This cmdlet is not to be used for setting the owner or owning group.")]
-        public DataLakeStoreItemAcl Acl { get; set; }
+        public DataLakeStoreItemAce[] Acl { get; set; }
```

Comment:
@begoldsm is this a breaking change? Are existing scripts that customers wrote that pass a `DataLakeStoreItemAcl` object to the `Acl` parameter now broken?